components/Downloads/ChangelogModal/index.tsx

```diff
@@ -0,0 +1,43 @@
+import * as Dialog from '@radix-ui/react-dialog';
+import type { FC, PropsWithChildren, ReactNode, ComponentProps } from 'react';
+
+import AvatarGroup from '@/components/Common/AvatarGroup';
+// TODO: Add styling
+// import styles from './index.module.css';
+import Link from '@/components/Link';
+
+type ChangelogModalProps = {
+  heading: string;
+  subheading: string;
+  avatars: ComponentProps<typeof AvatarGroup>['avatars'];
+  trigger: ReactNode;
+  children: ReactNode;
+};
+
+const ChangelogModal: FC<PropsWithChildren<ChangelogModalProps>> = ({
+  heading,
+  subheading,
+  avatars,
+  trigger,
+  children,
+}) => {
+  return (
```

Comment:
No need of an return statement and brackets {} here

Reply:
You can just do :
```tsx
const MyComponent = (props) => (
<Jsx>
</Jsx>
):
```
In react when you use return it's because you need logic like state ...